dnf-behave-tests/dnf/config-repos-overrides.feature

```diff
@@ -0,0 +1,154 @@
+@dnf5
+Feature: Repository overriding configuration files
+
+
+Background: Set repositories
+  Given I use repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-thirdparty-updates"
+    And I use repository "dnf-ci-fedora-updates" with configuration
+        | key     | value |
+        | enabled | 0     |
+    And I use repository "dnf-ci-thirdparty" with configuration
+        | key     | value |
+        | enabled | 0     |
+
+
+Scenario: Test the overrides in "/usr/share/dnf5/repos.override.d"
+  Given I create file "/usr/share/dnf5/repos.override.d/10-disable-all.repo" with
+      """
+      [*]
+      enabled=0
+      """
+    And I create file "/usr/share/dnf5/repos.override.d/20-enable-some.repo" with
+      """
+      [dnf-ci-fedora-updates]
+      enabled=1
+      """
+   When I execute dnf with args "repolist"
+   Then the exit code is 0
+    And stdout does not contain "dnf-ci-fedora\s+dnf-ci-fedora"
+    And stdout does not contain "dnf-ci-thirdparty-updates\s+dnf-ci-thirdparty-updates"
+    And stdout contains "dnf-ci-fedora-updates\s+dnf-ci-fedora-updates"
+    And stdout does not contain "dnf-ci-thirdparty\s+dnf-ci-thirdparty"
+
+
+Scenario: The conf file in "/etc/dnf/repos.override.d" hides the file of the same name in "/usr/share/dnf5/repos.override.d"
+  Given I create file "/usr/share/dnf5/repos.override.d/10-disable-all.repo" with
```

Comment:
Why do you need this condition for the test. It looks like that there are more rules in test scenario therefore the outcome might be a consequence of something else.

Reply:
@j-mracek I simplified this test.